### PR TITLE
Model, MovieDao, MovieService,HibernateUtil classes implemented

### PR DESCRIPTION
**Pull request by semyonich**

### CODE REVIEW

**Review by MykolaFatkullin — APPROVED** [review]

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @Override
+    public Movie add(Movie movie) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            Long movieId = (Long) session.save(movie);
+            transaction.commit();
+            movie.setId(movieId);
```
Comment (Sofasmile) [inline src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_01.html

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Unable to add " + movie.getTitle() + "to DB", e);
```
Comment (Sofasmile) [inline src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: let's create your custom exception

**Review by Sofasmile — CHANGES_REQUESTED** [review]

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java (hunk 1 above)
Comment (semyonich) [reply src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: done

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java (hunk 2 above)
Comment (semyonich) [reply src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**